cdap-api/src/main/java/co/cask/cdap/api/RuntimeContext.java

```diff
@@ -37,12 +37,17 @@
   Map<String, String> getRuntimeArguments();
 
   /**
-   * @return The application namespace
+   * @return The cluster name.
```

Comment:
No trailing period on Javadoc return statements, please. They are only used if the @return has another sentence that follows it.